lib/client.js

```diff
@@ -378,7 +370,17 @@ Object.assign(Client.prototype, {
       );
     }
     return unwrapRaw(value, true, builder, this, formatter) || '?';
-  },
+  }
+}
+
+Object.assign(Client.prototype, {
```

Comment:
This really feels like it should be composed in using an object rather than hacked in as a dynamically defined method, but that would be a larger, separate refactoring.